deps/openssl/openssl-cl_no_asm.gypi

```diff
@@ -16,6 +16,10 @@
       'includes': ['config/archs/linux-armv4/no-asm/openssl-cl.gypi'],
     }, 'target_arch=="arm64" and OS=="linux"', {
       'includes': ['config/archs/linux-aarch64/no-asm/openssl-cl.gypi'],
+    }, 'target_arch=="arm" and OS=="android"', {
+      'includes': ['config/archs/linux-armv4/no-asm/openssl-cl.gypi'],
+    }, 'target_arch=="arm64" and OS=="android"', {
```

Comment:
Is it possible to merge with line above as '... OS in "linux android" ...' ?

Reply:
Arg did not want to approve yet ...

Reply:
thanks, merged into linux condition.